bionic/aip/state.py

```diff
@@ -2,7 +2,9 @@
 
 
 class AipError(Exception):
-    pass
+    def __init__(self, job_id, error_message):
+        self.job_id = job_id
```

Comment:
When this exception is converted to a string, I think it will be rendered as a tuple: `(job_id, error_message)`. You probably want to provide a human-readable message, either by calling the superclass constructor (like `CodeVersioningError` does) or by implementing your own `__str__` method.

Reply:
Reverted changes to this class.